### PR TITLE
DOCS: Translate to Norwegian

### PR DESCRIPTION
**Pull request by 225559**
This is the start of translating `red/docs` to Norwegian

### CODE REVIEW

**Comment by greggirwin** [comment]
Sorn, thanks so much for your work. I'm sorry I haven't had time to review and merge things, but will try to catch up soon.

**Comment by maksumic** [comment]
> Sorn, thanks so much for your work. I'm sorry I haven't had time to review and merge things, but will try to catch up soon.

My pleasure! I will continue to work on it when I find time.

**Comment by 225559** [comment]
I do not have time to continue on this.

**Comment by greggirwin** [comment]
Do you feel it's in a mergeable state, so someone else can use it as a baseline for further work?

File: no/draw.adoc [hunk 1]
```diff
@@ -260,26 +260,25 @@ pen <color>
...
+<start>    : (valgfritt) startpunkt (pair!). 
+<end>      : (valgfritt om ikke <start>) sluttpunkt (pair!).
+<spread>   : (valgfritt) spredningsområde (word!).
 ----
 
-*Description*
+*Beskrivelse*
 
-Sets a linear gradient to be used for drawing operations. The following values are accepted for the spread method: `pad`, `repeat`, `reflect` (currently `pad` is same as `repeat` for Windows platform).
+Setter en lineær gradient som brukes til tegneoperasjonene. Følgende verdier aksepteres for spredningsmetoden: `pad`, `repeat`, `reflect` (for øyeblikket er `pad` the samme som `repeat` på Windows-plattformen).
```
Comment (225559) [inline no/draw.adoc:279]: Should be `det samme som` not `the samme som`

**Comment by 225559** [comment]
It is not a very good baseline because `draw.adoc` is unfinished.
But, ≈400 lines are translated with care.

**Comment by 225559** [comment]
I'm closing this PR.

**Comment by greggirwin** [comment]
Thanks for your efforts on this. I hope someone can make use of your work in the future.

**Pull request closed without merging**